static/styles/subscriptions.css

```diff
@@ -311,6 +311,13 @@
         a {
             color: inherit;
         }
+
+        br {
+            /* We only want the text to be shown in different lines
+            having line-height: 20px; set on it create huge spacing
+            between lines which looks very odd. */
+            line-height: 0;
+        }
```

Comment:
Can you post a screenshot to explain this?

Reply:
![image](https://user-images.githubusercontent.com/63504956/122973909-5db7db80-d3af-11eb-971f-029ca811bdb1.png)

Reply:
It turns out that it can be solved more elegantly without CSS changes if we do not wrap `{{message}}` in `p` tag as it has its own margin.
Image without this CSS change and without wrapping `{{message}}` with p.
![image](https://user-images.githubusercontent.com/63504956/122975536-116d9b00-d3b1-11eb-82bf-09b3168ce178.png)